File: CRM/src/app/shared/eHospital_DongNai_A/Sync_TransactionData_KeyGeneration.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sync_TransactionData_KeyGeneration } from './Sync_TransactionData_KeyGeneration.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sync_TransactionData_KeyGenerationService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ID', 'SiteCode', 'TableName', 'UpdatedDate'];

    List: Sync_TransactionData_KeyGeneration[] | undefined;
    ListFilter: Sync_TransactionData_KeyGeneration[] | undefined;
    FormData!: Sync_TransactionData_KeyGeneration;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sync_TransactionData_KeyGeneration";
    }
}

File: Service_eHospital_DongNai_A/Implement/Sync_TransactionData_KeyGenerationService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class Sync_TransactionData_KeyGenerationService : BaseService<Sync_TransactionData_KeyGeneration, ISync_TransactionData_KeyGenerationRepository>
    , ISync_TransactionData_KeyGenerationService
    {
    private readonly ISync_TransactionData_KeyGenerationRepository _Sync_TransactionData_KeyGenerationRepository;
    public Sync_TransactionData_KeyGenerationService(ISync_TransactionData_KeyGenerationRepository Sync_TransactionData_KeyGenerationRepository) : base(Sync_TransactionData_KeyGenerationRepository)
    {
    _Sync_TransactionData_KeyGenerationRepository = Sync_TransactionData_KeyGenerationRepository;
    }
    }
    }

